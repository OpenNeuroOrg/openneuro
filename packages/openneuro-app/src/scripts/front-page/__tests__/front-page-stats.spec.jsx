import React from 'react'
import { shallow } from 'enzyme'
import FrontPageStats from '../front-page-stats.jsx'

describe('FrontPageTopStats', () => {
  it('renders container correctly', () => {
    const wrapper = shallow(<FrontPageStats />)
    expect(wrapper).toMatchSnapshot()
  })
})
