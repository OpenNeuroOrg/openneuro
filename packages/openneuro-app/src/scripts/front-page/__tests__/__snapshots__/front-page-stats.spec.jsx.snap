// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrontPageTopStats renders container correctly 1`] = `
<Styled(section)
  className="openneuro-stats"
>
  <Styled(div)>
    <Styled(section)
      className="dataset-count"
    >
      <Styled(div)>
        <DatasetCount />
      </Styled(div)>
      <Styled(div)>
        Public Datasets
      </Styled(div)>
    </Styled(section)>
    <Styled(div) />
    <Styled(section)
      className="participant-count"
    >
      <Styled(div)>
        <ParticipantCount />
      </Styled(div)>
      <Styled(div)>
        Participants
      </Styled(div)>
    </Styled(section)>
  </Styled(div)>
</Styled(section)>
`;
